### PR TITLE
pie-microsite@v1.43.0 - Design Tokens - updates and maintenance foundations page

### PR DESCRIPTION
**Pull request by raoufswe**
- adds design tokens - updates and maintenance- page

### CODE REVIEW

File: apps/pie-microsite/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v1.43.0
```
Comment (jamieomaguire) [inline apps/pie-microsite/CHANGELOG.md:6]: can you add this to the PR description please? :)

Comment (raoufswe) [reply apps/pie-microsite/CHANGELOG.md:6]: 🚀

**Comment by jamieomaguire** [comment]
All looks good but I'm very surprised we're allowed to use that image of slack!

**Review by jamieomaguire — DISMISSED** [review]

**Comment by jamieomaguire** [comment]
Screenshot 2023-01-20 at 18 01 16

Small question for design - is the margin between the heading and end of the bullet list correct? (nothing to do with this PR but we can correct in a future one if not)

**Review by fernandofranca — APPROVED** [review]

**Review by fernandofranca — APPROVED** [review]

**Review by FayeCarter — APPROVED** [review]

**Pull request merged**